### PR TITLE
Create participant docRef

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-428

### CODE REVIEW

File: src/main/java/com/celements/course/service/CourseService.java [hunk 1]
```diff
@@ -55,6 +62,9 @@
   ModelContext modelContext;
 
   @Requirement
+  ModelUtils modelUtils;
```
Comment (fpichler) [inline src/main/java/com/celements/course/service/CourseService.java]: private?

File: src/main/java/com/celements/course/service/CourseService.java [hunk 2]
```diff
@@ -55,6 +62,9 @@
...
+
+  @Requirement
```
Comment (fpichler) [inline src/main/java/com/celements/course/service/CourseService.java:67]: private?

File: src/main/java/com/celements/course/service/CourseService.java [hunk 3]
```diff
@@ -64,8 +74,12 @@
   IMailSenderRole mailSender;
 
   @Requirement
+  ConfigurationSource cfgSrc;
```
Comment (fpichler) [inline src/main/java/com/celements/course/service/CourseService.java]: private?

**Review by fpichler — CHANGES_REQUESTED** [review]

**Comment by msladek** [comment]
@fpichler has approved

**Pull request merged**